test/bin/build_images.sh

```diff
@@ -25,6 +25,29 @@ osbuild_logs() {
     done
 }
 
+extract_container_images() {
+    local -r version=$1
+    local -r repo_url=$2
+    local -r outfile=$3
+
+    mkdir -p "${IMAGEDIR}/release-info-rpms"
+    pushd "${IMAGEDIR}/release-info-rpms"
+    sudo bash -c 'cat > /etc/yum.repos.d/microshift.repo' << EOF
+[microshift]
+name="microshift ${version}"
+baseurl="${repo_url}"
+enabled=1
+check_gpg=false
+EOF
+    sudo dnf download microshift-release-info
```

Comment:
Should we not use `--repo` argument here to specify which repo to use for download of the package? Otherwise, it'll always get the latest one, I think

Reply:
Improved it.